exposing-to-internet/README.md

```diff
@@ -0,0 +1,88 @@
+# Exposing Apigee to the Internet
+
+This sample shows how to expose an Apigee instance to the internet using an [external HTTP(S) load balancer](https://cloud.google.com/load-balancing/docs/https) and [Private Service Connect](https://cloud.google.com/apigee/docs/api-platform/system-administration/northbound-networking-psc).
+
+## How it works
+
+With Apigee X customers have full control over whether or not to expose their [runtime](https://cloud.google.com/apigee/docs/api-platform/get-started/what-apigee#componentsofapigeeedge-edgeapiservices) instances externally. Apigee X instances are not exposed to the internet by default, however customers may choose to serve traffic to external API consumers by placing an external HTTP(S) load balancer in front of Apigee. Customers may then leverage other features of Google Cloud Load Balancing such as [Cloud Armor](https://cloud.google.com/armor) WAF & DDoS protection for additional security in front of their APIs.
+
+When following the Apigee X [provisioning wizard](https://cloud.google.com/apigee/docs/api-platform/get-started/wizard-select-project), you will be prompted to [configure access routing](https://cloud.google.com/apigee/docs/api-platform/get-started/configure-routing) for your newly created instance. If you choose the internal option, the instance is only accessible internally via your GCP VPC network. If you subsequently decide you wish to expose it externally, this sample shows you how to do so.
```

Comment:
`If you subsequently decide you wish to expose it externally, this sample shows you how to do so` needs correction